linkerd/protocol/benchmark/src/main/scala/io/buoyant/protocol/AccessLoggerBenchmark.scala

```diff
@@ -0,0 +1,34 @@
+package io.buoyant.linkerd.protocol
+
+import com.twitter.finagle.Service
+import com.twitter.finagle.http._
+import com.twitter.util.Future
+import com.twitter.util.StdBenchAnnotations
+import com.twitter.logging.NullLogger
+import org.openjdk.jmh.annotations._
+import io.buoyant.linkerd.protocol.http.AccessLogger
+import io.buoyant.test.Awaits
+
+// ./sbt 'project linkerd-protocol-benchmark' jmh:run -i 20 -prof gc .*AccessLoggerBenchmark.*
```

Comment:
Can you change this to:

    ./sbt 'project linkerd-protocol-benchmark' 'jmh:run -i 20 -prof gc .*AccessLoggerBenchmark.*'